cloud_infrastructure_access/rosa-private-cluster.adoc

```diff
@@ -7,7 +7,7 @@ toc::[]
 
 A {product-title} cluster can be made private so that internal applications can be hosted inside a corporate network. In addition, private clusters can be configured to have only internal API endpoints for increased security.
 
-// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager* (OCM).
+// {product-title} administrators can choose between public and private cluster configuration from within the *OpenShift Cluster Manager*.
```

Comment:
Please remove 'the' so it's consistent with how other docs refer to is as "OpenShift Cluster Manager"